docs/preview_upgrade_instructions.md

```diff
@@ -1,59 +1,112 @@
-    - Use the NuGet package manager to uninstall the NuGet package `Microsoft.VCRTForwarders.140` from the main app package.
-
-    - Open the .vcxproj in the main app project
-        - Locate the following XML element:
-          ```xml
-          <PropertyGroup Label="Configuration">
-          ...
-          </PropertyGroup>
-          ```
-          and add the following line of XML to its body (this sets the MSBuild property `DesktopCompatible` to `true`):
-          ```xml
-          <DesktopCompatible>true</DesktopCompatible>
-          ```
-
-    - Open the .wapproj in the associated Windows Application Packaging Project and delete the following lines:
-      ```xml
-      <ItemGroup>
-        <SDKReference Include="Microsoft.VCLibs.Desktop, Version=14.0" />
-        <!-- Needed for ucrtbased.dll when running a debug build. -->
-        <SDKReference Include="Microsoft.VCLibs, Version=14.0" Condition="'$(Configuration)' == 'Debug'" />
-      </ItemGroup>
-      ```
-
-    - Create a new C++ “Blank App, Packaged (WinUI in Desktop)” project, locate the folder containing the associated Windows Application Packaging Project, and copy the contents of the `“build\”` subdirectory (`Microsoft.WinUI.AppX.targets`) into the corresponding `“build\”` subdirectory of your app’s associated Windows Application Packaging Project, overwriting any existing files when prompted.
+## Upgrading your WinUI 3 app from Preview 4 to Project Reunion 0.5 Preview 
+
+This is written for a developer who has an existing WinUI 3 Preview 4 app and wants to upgrade it to use WinUI 3 - Project Reunion 0.5 Preview. 
+
+Before starting, make sure you have all the WinUI 3 - Project Reunion 0.5 Preview pre-requisites installed, including the Project Reunion VSIX and NuGet package. For installation instructions, see [this article](https://docs.microsoft.com/en-us/windows/apps/project-reunion/#set-up-your-development-environment). 
+
+### If your app is a C# Blank App (UWP):
+
+1. Go to Tools->Nuget Package Manager-> Package Manager Console in Visual Studio
+2. Type in ```uninstall-package Microsoft.WinUI -ProjectName {yourProject}```
+3. Type in ```install-package Microsoft.ProjectReunion -Version 0.5.0-prerelease -ProjectName {yourProject}```
+
+
+### If your app is a C# Blank App Packaged (WinUI in Desktop):
+
+Go to Tools->Nuget Package Manager-> Package Manager Console in Visual Studio
+
+  1. Right Click your Application (Package) folder in the solution explorer and select "unload project"
+  2. Type in ```uninstall-package Microsoft.WinUI -ProjectName {yourProject}```
+  3. Type in ```install-package Microsoft.ProjectReunion -Version 0.5.0-prerelease -ProjectName {yourProjectName}```
+  4. Type in ```install-package Microsoft.ProjectReunion.Foundation -Version 0.5.0-prerelease -ProjectName {yourProjectName}```
+  5. Type in ```install-package Microsoft.ProjectReunion.WinUI -Version 0.5.0-prerelease -ProjectName {yourProjectName}```
+  6. Reload the (package) folder
```

Comment:
For the same reasons, you can also delete this line since the -ProjectName parameter solves this.